integration-tests/test/fee-payment.spec.ts

```diff
@@ -3,20 +3,44 @@ import chaiAsPromised from 'chai-as-promised'
 chai.use(chaiAsPromised)
 
 /* Imports: External */
-import { BigNumber, Contract, utils } from 'ethers'
-import { TxGasLimit, TxGasPrice } from '@eth-optimism/core-utils'
+import { ethers } from 'hardhat'
+import {
+  BigNumber,
+  Contract,
+  utils,
+  Wallet,
+  constants,
+  ContractFactory,
+} from 'ethers'
+import {
+  TxGasLimit,
+  TxGasPrice,
+  LibEIP155TxStruct,
+} from '@eth-optimism/core-utils'
 import { predeploys, getContractInterface } from '@eth-optimism/contracts'
 
 /* Imports: Internal */
 import { OptimismEnv } from './shared/env'
 import { Direction } from './shared/watcher-utils'
 
+// TODO: import me from core-utils via kevin's incoming PR
```

Comment:
todo reminder

Reply:
This should be imported from a test package instead of `core-utils`? Its a dummy tx used for testing purposes